src/execute.c

```diff
@@ -1343,3 +1346,13 @@ jv jq_get_error_message(jq_state *jq)
 {
   return jv_copy(jq->error_message);
 }
+
+int jq_get_parser_maxdepth(jq_state *jq)
+{
+  return jq->parser_maxdepth;
+}
+
+void jq_set_parser_maxdepth(jq_state *jq, int parser_maxdepth)
+{
+  jq->parser_maxdepth = parser_maxdepth;
+}
```

Comment:
Missing new lines?

Reply:
Whoops, added it on in 586e96d! (Would you want a PR that adds this as a CI lint?)

Reply:
Maybe would be good, i like linters. Let's see what the other maintainers think